values/prometheus-operator/prometheus-operator-team.gotmpl

```diff
@@ -22,9 +23,10 @@ defaultRules:
     kubeStateMetrics: true
     network: false
     node: false
+    nodeExporterAlerting: true
```

Comment:
Why does prometheus for team needs to enable `nodeExporterAlerting`, `nodeExporterRecording`